src/core/transform.js

```diff
@@ -125,10 +125,12 @@ import p5 from './main';
  *   let st = sin(rad);
  *   // Matrix for rotation around the Y axis
  *   // prettier-ignore
```

Comment:
Just a heads up, this `prettier-ignore` has been left in, in case you wanted to remove it or replace it with an equivalent!